meta-luneos/recipes-connectivity/ofono/ofono-halium_1.29.bb

```diff
@@ -11,6 +11,7 @@ DEPENDS = "dbus glib-2.0 udev mobile-broadband-provider-info ell"
 COMPATIBLE_MACHINE = "^halium$"
 # This recipe acts as a replacement for the ofono recipe
 PROVIDES += "ofono"
+RPROVIDES:${PN} += "ofono"
```

Comment:
With this new version based on the preferred provider, I'm not sure we should override this "PROVIDES" and "RPROVIDES" anymore

Reply:
Seems we still need both, without won't build.